extensions/smallrye-graphql-client/runtime/src/main/java/io/quarkus/smallrye/graphql/client/runtime/GraphQLClientConfig.java

```diff
@@ -33,6 +33,20 @@
     @ConfigItem(defaultValue = "graphql-transport-ws")
     public Optional<List<String>> subprotocols;
 
+    /**
+     * If true, then queries and mutations will run over the websocket transport rather than pure HTTP.
+     * Off by default, because it has higher overhead.
+     */
+    @ConfigItem
+    public Optional<Boolean> executeSingleResultOperationsOverWebsocket;
+
+    /**
+     * Maximum time in milliseconds that will be allowed to wait for the server to acknowledge a websocket connection
+     * (send a subprotocol-specific ACK message).
+     */
+    @ConfigItem
+    public Optional<Integer> websocketInitializationTimeout;
```

Comment:
Use `OptionalInt` instead

Reply:
Or maybe `Duration`?

Reply:
For `Duration`: The  counterpart in SmallRye API uses a simple number in milliseconds so I'd like to have a 1:1 mapping between Quarkus and SmallRye properties... And this will likely be a small number so no need to allow easily setting it to minutes etc. 

For `OptionalInt`: And some other properties in here are `Optional<Integer>` so I would rather retain consistency, unless we want to change all of them to `OptionalInt`.

Reply:
+1 to change all to `OptionalInt`, that's what we do in other extensions too

Reply:
Ok, done that :)